internal/monitors/cadvisor/converter/converter.go

```diff
@@ -11,6 +11,13 @@ import (
 	info "github.com/google/cadvisor/info/v1"
 	"github.com/signalfx/golib/datapoint"
 	"github.com/signalfx/signalfx-agent/internal/utils"
+	stats "k8s.io/kubernetes/pkg/kubelet/apis/stats/v1alpha1"
+)
+
+const (
+	kubernetesPodUID    = "kubernetesPodUID"
```

Comment:
These string values aren't right.  They are supposed to be `_` separated.

Reply:
Also I'm not a huge fan of consting something that is only used twice and adds a level of indirection to figure out what the dim name really is.  I would just use string literals at the map set point.

Reply:
Replaced to using string literals.